animation_nodes/nodes/mesh/get_vertex_color_layer.py

```diff
@@ -0,0 +1,60 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types import AnimationNode
+from ... data_structures import Color, ColorList, VirtualColorList
+from .. color.c_utils import getVertexColorsFromLoopColors, getPolygonColorsFromLoopColors
+
+colorModeItems = [
+    ("LOOP", "Loop", "Get color of every loop vertex", "NONE", 0),
+    ("VERTEX", "Vertex", "Get color of every vertex", "NONE", 1),
+    ("POLYGON", "Polygon", "Get color of every polygon", "NONE", 2)    
+]
+
+class GetVertexColorLayerNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GetVertexColorLayerNode"
+    bl_label = "Get Vertex Color Layer"
+    bl_width_default = 155
+    errorHandlingType = "EXCEPTION"
+
+    colorMode: EnumProperty(name = "Color Mode", default = "LOOP",
+        items = colorModeItems, update = propertyChanged)
+
+    def create(self):
+        self.newInput("Mesh", "Mesh", "mesh", dataIsModified = True)
+        self.newInput("Text", "Name", "colorLayerName")
+
+        self.newOutput("Color List", "Colors", "colors")        
+
+    def draw(self, layout):
+        layout.prop(self, "colorMode", text = "")
+
+    def execute(self, mesh, colorLayerName):
+        if mesh is None: return ColorList()
+        if colorLayerName == "":
+            self.raiseErrorMessage("No Vertex Color Layer Name.")
+        
+        defaultColor = Color((0, 0, 0, 1))
+        colorsList = ColorList(length = len(mesh.polygons.indices))
+        
+        if mesh.getVertexColors(colorLayerName) is not None:
+            colorsList = mesh.getVertexColors(colorLayerName)
+        else:
+            self.raiseErrorMessage(f"No Vertex Color Layer of Name '{colorLayerName}'."
+                                   f" Layers: {', '.join(mesh.getVertexColorLayerNames())}")
```

Comment:
This might be a little long for an error message. Also, `of name` -> `with name`.

Reply:
I added the `f" Layers: {', '.join(mesh.getVertexColorLayerNames())}"` to avoid guesswork or to check names with Viewer Node. Should I remove this line to make the error message short?

Reply:
Yes. The Mesh repr method already provide those names.

Reply:
Now it looks like this `self.raiseErrorMessage(f"Mesh doesn't has '{colorLayerName}' Vertex Color Layer.")`. is it fine?

Reply:
I think this is better:

> Mesh doesn't have a vertex color layer with the name '{colorLayerName}'.

Reply:
Okay, here it is
![Screenshot from 2019-12-23 00-24-14](https://user-images.githubusercontent.com/30294746/71325992-a5d81280-251a-11ea-8365-80091b7312fd.png)

Reply:
Looks good.